projects/1.20/assets/productivebees/productivebees/patchouli_books/guide/zh_cn/entries/bees/wild/magmatic.json

```
@@ -0,0 +1,18 @@
+{
+    "name": "熔岩蜜蜂",
```

```suggestion
    "name": "岩浆蜜蜂",
```
记得全文件通查，已经在其他文件里看见过错误位置了